apollo-core/src/test/java/com/ctrip/framework/foundation/internals/UtilsTest.java

```diff
@@ -0,0 +1,89 @@
+package com.ctrip.framework.foundation.internals;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * @author Josh Knopp(https://github.com/joshknopp)
+ */
+class UtilsTest {
+    @Nested
+    class IsBlank {
+        @Test
+        void trueGivenNull() {
+            assertTrue(Utils.isBlank(null));
+        }
+
+        @Test
+        void trueGivenEmpty() {
+            assertTrue(Utils.isBlank(""));
+        }
+
+        @Test
+        void trueGivenWhitespace() {
+            assertTrue(Utils.isBlank("   "));
+        }
+
+        @Test
+        void falseGivenLoremIpsum() {
+            String value = "Lorem Ipsum";
+            assertFalse(Utils.isBlank("Lorem Ipsum"));
+        }
+
+        @Test
+        void falseGivenWhitepsacePadded() {
+            assertFalse(Utils.isBlank("   Lorem Ipsum   "));
+        }
+    }
+
+    @Nested
+    class IsOsWindows {
+        private String actualOsName;
+
+        @BeforeEach
+        void setUp() {
+            actualOsName = System.getProperty("os.name");
+        }
```

Comment:
How about delete the setUp method?
Just assign value to `actualOsName` once.

```suggestion
        private final String actualOsName = System.getProperty("os.name");
```

Reply:
Thanks @Anilople for the feedback!